Mathlib/RingTheory/WittVector/Truncated.lean

```diff
@@ -213,17 +212,26 @@ instance hasNatPow : Pow (TruncatedWittVector p n R) ℕ :=
 
 @[simp]
 theorem coeff_zero (i : Fin n) : (0 : TruncatedWittVector p n R).coeff i = 0 := by
-  show coeff i (truncate_fun _ 0 : TruncatedWittVector p n R) = 0
-  rw [coeff_truncate_fun, WittVector.zero_coeff]
+  show coeff i (truncateFun _ 0 : TruncatedWittVector p n R) = 0
+  rw [coeff_truncateFun, WittVector.zero_coeff]
 #align truncated_witt_vector.coeff_zero TruncatedWittVector.coeff_zero
 
 end TruncatedWittVector
 
-/- ./././Mathport/Syntax/Translate/Expr.lean:336:4: warning: unsupported (TODO): `[tacs] -/
-/-- A macro tactic used to prove that `truncate_fun` respects ring operations. -/
-unsafe def tactic.interactive.witt_truncate_fun_tac : tactic Unit :=
-  sorry
-#align tactic.interactive.witt_truncate_fun_tac tactic.interactive.witt_truncate_fun_tac
+/-- A macro tactic used to prove that `truncateFun` respects ring operations. -/
+macro (name := wtf_tac) "wtf_tac" : tactic =>
```

Comment:
Please rename this to `witt_truncateFun_tac`.

Reply:
Done.